src/questions/chapter5.yaml

```diff
@@ -0,0 +1,1239 @@
+  answerKeys:
+    - A
+  topic: OOP
+  subTopics:
+  difficultyLevel: Medium
+  choices:
+    - content: (a), (b) and (c)
+      key: A
+      order: 0
+    - content: (b), (c) and (d)
+      key: B
+      order: 1
+    - content: (a), (d) and (e)
+      key: C
+      order: 2
+    - content: (c), (d) and (e)
+      key: D
+      order: 3
+
+- uuid: 46a72ca3-f513-417c-bf3e-099c9610f7e9
+  type: MultipleChoice
+  content: Which objects can only be accessed internally from the current contract instances?
+  hint: noHint
+  explanation: Private objects can only be accessed internally from the current contract instances.
+  answerKeys:
+    - B
+  topic: Encapsulation
+  subTopics:
+  difficultyLevel: Low
+  choices:
+    - content: public
+      key: A
+      order: 0
+    - content: private
+      key: B
+      order: 1
+    - content: internal
+      key: C
+      order: 2
+    - content: external
+      key: D
+      order: 3
+
+- uuid: 5d6b3cca-0774-4bea-90bc-c0fb6bb0c0c0
+  type: MultipleChoice
+  content: Which of the following does not belong to state visibility?
+  hint: hint
+  explanation:
+  answerKeys:
+    - D
+  topic: Encapsulation
+  subTopics:
+    - state visibility
+  difficultyLevel: Low
+  choices:
+    - content: Public
+      key: A
+      order: 0
+    - content: Internal
+      key: B
+      order: 1
+    - content: Private
+      key: C
+      order: 2
+    - content: External
+      key: D
+      order: 3
+
+- uuid: 6fdde9f4-9853-413c-a110-7c4ee6c8aabb
+  type: MultipleChoice
+  content: The process of defining multiple contracts related to each other through parent-child relationships is called?
+  hint: noHint
+  explanation:
+  answerKeys:
+    - C
+  topic: Inheritance
+  subTopics:
+    - Inheritance
+  difficultyLevel: Low
+  choices:
+    - content: Abstraction
+      key: A
+      order: 0
+    - content: Encapsulation
+      key: B
+      order: 1
+    - content: Inheritance
+      key: C
+      order: 2
+    - content: None
+      key: D
+      order: 3
+
+- uuid: 79ff51fb-9703-44ea-bf0d-a092cea80e17
+  type: MultipleChoice
+  content: Solidity uses to force a specific order in graphs of base contracts is known as 
+  hint: noHint
+  explanation:
+  answerKeys:
+    - C
+  topic: Inheritance
+  subTopics:
+  difficultyLevel: Medium
+  choices:
+    - content: C1 Linearization
+      key: A
+      order: 0
+    - content: Method Revolution Order
+      key: B
+      order: 1
+    - content: C3 Linearization
+      key: C
+      order: 2
+    - content: None
+      key: D
+      order: 3
+
+- uuid: 3c8b8711-b120-46db-9bc8-46d6fa292dae
+  type: MultipleChoice
+  content: Which of the following inheritance does not belongs to solidity?
+  hint: noHint
+  explanation:
+  answerKeys:
+    - B
+  topic: Inheritance
+  subTopics:
+    - Inheritance types
+  difficultyLevel: Medium
+  choices:
+    - content: Single Inheritance
+      key: A
+      order: 0
+    - content: Hybrid Inheritance
```

Comment:
`Hybrid Inheritance` and `Hierarchical Inheritance` are not that famous. If we want to keep this questions, we should definition against each choice